cmd/certificates/watcher.go

```diff
@@ -88,7 +88,7 @@ func (watcher *CertificateWatcher) updateCertificatesFromSecret() (bool, error)
 		}
 	}
 
-	isValid, err := certsutils.ValidateCertificateExpiration(secret.Data[certificates.ServerCert], certificateRenewalInterval, time.Now(), log)
+	isValid, err := certsutils.ValidateCertificateExpiration(secret.Data[certificates.ServerCert], certificateRenewalInterval, time.Now(), log.Logger)
```

Comment:
this is our code, we could expect our logger (https://github.com/Dynatrace/dynatrace-operator/blob/9a3bcd1bc6b3eb9f946d056c83474efb23dde74b/pkg/util/certificates/certificates.go#L11-L11)